style.css

```diff
@@ -0,0 +1,397 @@
+	width: auto;
+}
+.left-menu {
+	display: flex;
+	float: left;
+	align-items: center;
+}
+.right-menu {
+	display: flex;
+	float: right;
+	align-items: center;
+}
+.center-menu {
+	height: 40px;
+	display: flex;
+	float: left;
+	align-items: center;
+}
+.apps-icon {
+	height: 22px;
+}
+.settings-icon {
+	height: 22px;
+}
+/* ------------------->END OF HEADER SECTION^<--------------------  */
+
+
+/* ------------------->START OF SECTION<--------------------  */
+#void-heading {
+	display: none;
+}
+#video_wrapper {
+	grid-column: 1 / span 2;
+position: relative;
+padding-bottom: 56.25%; /* 16:9, for an aspect ratio of 1:1 change to this value to 100% */
+}
+iframe{
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+main {
+	border-top: 15px solid rgb(236, 236, 236);
+	;
+	border-bottom: 5px solid rgb(236, 236, 236);
+	;
+	margin-top: 40px;
+	display: grid;
+	grid-template-columns: 2fr min-content;
+	grid-gap: 15px;
+	margin-left: -10px;
+	margin-right: -10px;
+	font-family: arial;
+}
+.left-section {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-rows: min-content 100px 60px 30px 200px 40px 100px 1fr;
+	grid-column-start: 0;
+	grid-column-end: 1;
+	border-right: 10px solid rgb(236, 236, 236);
+	;
+}
+.text-image-container {
+	display: flex;
+	align-items: center;
+}
+#video-iframe {
+	width: 100%;
+	height: 100%;
+	grid-column: 1 / span 2;
+}
+.video-title {
+	margin-top: 20px;
+	font-size: 25px;
+	grid-column: 1 / span 2;
+	border-top: 13px solid rgb(236, 236, 236);
+	;
+	font-family: arial;
+	color: black;
+	opacity: 0.9;
+	padding-top: 10px;
+	padding-left: 15px;
+}
+.profile-icon-section {
+	display: grid;
+	grid-column: 1 / span2;
+	grid-gap: 5px;
+	grid-template-columns: min-content 3fr auto;
+	padding-top: 5px;
+	padding-left: 15px;
+}
+.icon-element {
+	place-self: center;
+}
+.views-element {
+	margin-top: 30px;
+	place-self: center;
+	font-size: 20px;
+	color: gray;
+	font-family: arial;
+}
+.username {
+	margin-bottom: 5px;
+	font-size: 15px;
+	font-family: arial;
+	color: black;
+}
+#youtube-subscribe-img {
+	width: auto;
+	height: 25px;
+	margin: 0px;
+}
+.viewers {
+	display: flex;
+	border: 1px solid rgb(236, 236, 236);
+	;
+	height: 22px;
+	font-size: 10px;
+	font-family: arial;
+	color: gray;
+	margin-left: -1px;
+	align-items: center;
+	padding-right: 8px;
+	padding-left: 8px;
+}
+.username-subscribe-button {
+	align-self: center;
+}
+#profile-icon-image {
+	height: 48px;
+	width: 48px;
+}
+.thumbs-up-section {
+	grid-row-start: 4;
+	grid-row-end: 5;
+	grid-column: 1 / span2;
+	padding-left: 5px;
+	display: flex;
+	justify-content: space-between;
+	padding-right: 5px;
+	font-size: 12px;
+	color: gray;
+	font-family: arial;
+	padding-top: 2px;
+	padding-left: 10px;
+}
+.thumbs-up-section img {
+	padding-left: 5px;
+	padding-right: 4px;
+	width: 22px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+}
+#plus-sign-icon-image {}
+
```

Comment:
Do not use empty rulesets CSS, line number 231 better to be deleted

Reply:
Thank you very much Anna, we're now working on these issues.